samples/sqltransport/native-integration/sample.md

```diff
@@ -0,0 +1,85 @@
+---
+title: SQL Server
+summary: 'How to integrate SQL Server transport with NHibernate persistence without outbox'
+tags:
+- SQL Server
+- NHibernate
+---
+
+ 1. Make sure you have SQL Server Express installed and accessible as `.\SQLEXPRESS`. Create a databases `samples`
+ 2. Start the Receiver project.
+ 3. In the Sender's console you should see `Press any key to send a message. Press `q` to quit` text when the app is ready. 
+ 4. Hit any key.
+ 5. A message will be sent using ADO.net and be received by the app.
+ 6. Open SQL Server Management Studio and go to the `samples` database.
+ 7. Open the Scripts.sql included in the sample
+ 7. Run the `SendFromTSQL` statement
+ 8. Notice how the app shows that a new message has been processed
+ 9. Create the `Orders` table using the `CreateLegacyTable` statement
+ 10. Create the insert trigger using the `CreateTrigger` statement
+ 11. Right click the table you just created and do `Edit top X rows`
+ 12. Notice that a message is received by the app for each "order" you create
+ 
+
+## Code walk-through
+
+The first thing you need to do when doing native integration with the SqlServer transport is figure out where to insert your "message". The database and server names can easily be found by looking at the connection string and the and the table name is by default the same as your endpoint name. So with the following endpoint configuration
+
+
+<!-- import EndpointConfiguration-->
+
+
+The table would be `Samples.SqlServer.NativeIntegration` in the database `samples` on server `.\SQLEXPRESS` (localhost). Now that we know where to put the data lets see how we can make sure that the endpoint can parse our message.
+
+### Serialization
+
+In this sample we'll be using JSON to serialize the messages but XML would have worked equally well. First we'll setup the endpoint to user JSON using the following configuration:
+
+<!-- import SerializerConfiguration-->
+
+Not that our endpoint can understand JSON we define a message contract to use. In NServiceBus messages are just C# classes so in this case we create the following class
+
+<!-- import MessageContract-->
+
+The final piece of the puzzle to to tell the serializer what C# class our JSON payload belongs to. We do this using the JSON.net `$type` attribute. The message body will then look as follows:
+
+<!-- import MessagePayload-->
+
+
+### Sending the message
+Not that we've done all the legwork send a message to our endpoint using plain ADO.net is as easy as
+
+<!-- import SendingUsingAdoNet-->
+
+Armed with this you will now be able to send messages from any app in your organization that supports ADO.net

```

Comment:
Armed with this now you will be able to send messages, in your organization, from any app that supports ADO.net.